content/pages/volunteers.md

```diff
@@ -0,0 +1,103 @@
+* **You will need:** A camera of your own or one that you can borrow.
+* People who are not willing to be photographed will be in lanyards so make sure that you don’t photograph them!
+* Photographs should be sent to the committee as soon as possible after
+  the event is over.
+* **Commitment:** Full length of the event if it is under 3 hours long. For events that run longer than that (such
+  as full day workshops), we will work out a shift system. Each shift will be under 3 hours.
+
+### Poster Posters
+
+* We will need 1-2 volunteers per event.
+* Notice will be given 4-5 days before the event.  You will have
+  to coordinate with the director of publicity to pick up printed posters,
+  supplies for putting them up, and deciding on locations to put them up.
+* **Commitment:** **Low:** 1-1.5 hours/event, can do them at your own pace.
+
+### Mentor for Workshops or Technical Events
+* Depending on the size of the event, we will need 3-7 volunteers per event.
+  Notice for this will be given 14-10 days before the event, with an explanation of the content being covered
+  and the difficulty level.
+* **You will need:**
+    - Enthusiasm for the topic of the workshop!
+    - Enough expertise in the material being presented or taught to help the
+      attendees of the workshop/event if they’re stuck or have questions.
+* **Commitment**: This is a higher commitment position than the previous
+  positions. In addition to attending the event, you will also have to spend
+  time familiarizing yourself with the material prior to it.
+
+### Systems Committee Volunteer
+
+* This is a great way to learn about Linux systems administration, version
+  control, and static website development!
+* You’ll work with our WiCS systems committee and help maintain our machines, track issues, etc.
+* Indicate your interest in being a systems-committee volunteer in the form above. Doing so will add
+  you to the wics-sys mailing list.
+* **You will need:** to be willing to work independently, try new things, follow along with existing work, and ask
+  questions. Prior experience, although appreciated, is not a pre-requisite.
+
+### Run an event with us!
+
+* **You** have to reach out to us [by email](mailto:wics-ugrad@lists.uwaterloo.ca) with an event that you’d like to
+  see or run on your own! If we approve it, we will help you run the event. In the email, make sure to include:
+    - name of the event 
+    - who is organizing or leading the event
+    - if it is a women only event
+    - a brief description of the event
+    - a tentative date and time for the event
+    - an estimated budget breakdown, if you require one
+* You’ll play a major role in running and organizing the event.
+* We’ll help you advertise it, book rooms for it, order food for it under our budget.
+* **Commitment:** 5-6 hours spread out over around 10 days.
+* Here are some examples of what you can do:
+    - Bring in a potential speaker to give a talk.
```

Comment:
nit: can you start this list with lower case and not end in periods, like the list directly above it?